WcaOnRails/app/views/admin/_nav.html.erb

```diff
@@ -6,17 +6,16 @@
       <% [
         { text: "List competitions", path: competitions_view, fa_icon: "list" },
         { text: "Delegates", path: delegates_stats_path, fa_icon: "sitemap" },
-        { text: "Delegate Probations", path: delegate_probations_path, fa_icon: "sitemap" },
         { text: "Edit teams", path: teams_path, fa_icon: "users" },
-        { text: "Posting Dashboard", path: results_posting_dashboard_path, fa_icon: "user lock" },
+        { text: "Posting Dashboard", path: panel_wrt_path + "#posting-dashboard", fa_icon: "user lock" },
```

Comment:
Rails URL helpers can do this without manually glueing together strings:
`panel_wrt_path(anchor: 'posting-dashboard')`

Reply:
cf. https://stackoverflow.com/a/7052410

Reply:
Oh that's interesting, done. Thanks.